### PR TITLE
Updated to Elm 0.18

### PR DESCRIPTION
**Pull request by hakonrossebo**
Hi, I've updated the code to work with Elm 0.18 and the updated version of Kwarrtz/render

### CODE REVIEW

File: src/Update.elm [hunk 1]
```diff
@@ -277,7 +277,7 @@ shipSpeed ship =
 isShipLanded : Ship -> Float -> Bool
 isShipLanded ship platformPos =
     abs ship.y
-        > 0.99
+        > 0.97
```
Comment (teh) [inline src/Update.elm:280]: Is this for nicer gameplay?

File: src/Update.elm [hunk 2]
```diff
@@ -335,15 +333,13 @@ isHit ( shipX, shipY ) ( newX, newY ) ( hitSoFar, ( prevX, prevY ) ) =
 
 toScreenCoords : ( Int, Int ) -> ( Float, Float ) -> ( Float, Float )
 toScreenCoords ( w, h ) ( x, y ) =
-    ( (x - 0.5) * (toFloat w), (y - 0.5) * (toFloat h) )
+    ( (x ) * (toFloat w), (y ) * (toFloat h) )
```
Comment (teh) [inline src/Update.elm:336]: Is this related to the renderer update?

**Review by teh — APPROVED** [review]
As always: many thanks! Apologies for the delay, this got lost in the past-holiday mail deluge :)

I approved, only have two questions for my own understanding.

File: src/Update.elm (hunk 1 above)
Comment (hakonrossebo) [reply src/Update.elm:280]: Yes, the ship was going almost below the platform before stopping so I had to adjust. Could not verify as I did not have the previous version running.

File: src/Update.elm (hunk 2 above)
Comment (hakonrossebo) [reply src/Update.elm:336]: Yes, the renderer changed to having origin at top left.
